### ISSUE TITLE
Can not parse name with a peroids (.)

### ISSUE THREAD

**Issue by wenjie1991** [raku-community-modules/XML#54]
It failed to parse the name with a periods in it.
```
my $xml-part = "<name_contain.periods>some text</name_contain.periods>";
my $doc = XML::Grammar.parse($xml-part);
# return (Any) in $doc
```
I tried to change token in the grammar to:
```
token pident {
  <!before \d> [ \d+ <.ident>* || <.ident>+ ]+ % <[-.]>
}
```
It worked when I defined the grammar in the script.

**Comment by adamsmd** [comment]
This should now be fixed by the now merged #62.  I think it can now be closed.

**Comment by wenjie1991** [comment]
Thanks! Nice to see this module come back to life :-)
